ToDoApp/Controllers/MainViewController.swift

```diff
@@ -6,46 +6,89 @@
 //
 import UIKit
 
-class MainViewController: UIViewController {
 
+
+final class MainViewController: UIViewController, AddItemDelegate {
+    // TableView에 사용될 데이터 모델
+    var items = [Info]()
+    
+    // MARK: - tableView 설정
+    let tableView: UITableView = {
+        let tableView = UITableView()
+        tableView.rowHeight = 120
+        return tableView
+    }()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
         
-        self.title = "홈"
+        self.title = "ToDoList"
         self.view.backgroundColor = .white
+    
+  
+        // TableView 설정
+        tableView.register(CustomCell.self, forCellReuseIdentifier: CustomCell.cellId)
+        tableView.delegate = self
+        tableView.dataSource = self
+        view.addSubview(tableView)
         
-        // 네비게이션 아이템 추가
-        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "person.fill"), style: .plain, target: self, action: #selector(goToProfileVC))
+        // TableView Layout 설정
+        tableView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top)
+            make.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom)
+            make.leading.equalTo(view.safeAreaLayoutGuide.snp.leading)
+            make.trailing.equalTo(view.safeAreaLayoutGuide.snp.trailing)
+        }
         
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "message.fill"), style: .plain, target: self, action: #selector(goToMessageVC))
+        // MARK: - navigationbar 설정
+        navigationItem.leftBarButtonItem = UIBarButtonItem(title: "Log out", style: .plain, target: self, action: #selector(goToLoginVC))
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "plus"), style: .plain, target: self, action: #selector(addTableView))
         
-                
+      
+        // 데이터 모델 초기화
+        items = [
+            Info(title:"1",priority: "High", memo: "item 1", date: "2023-04-28"),
+            Info(title:"2",priority: "Medium", memo: "item 2", date: "2023-04-29"),
+            Info(title:"3",priority: "Low", memo: "item 3", date: "2023-04-30")
+        ]
     }
-    
-    
-    @objc fileprivate func goToProfileVC(){
-        print("MainViewController - goToProfileVC() 호출됨")
-        
-        let profileVC = LoginViewController()
-        self.navigationController?.pushViewController(profileVC, animated: true)
-        profileVC.modalPresentationStyle = .fullScreen
-//        self.present(profileVC, animated: true)
-        
+    // MARK: - 메서드 설정
+    @objc fileprivate func goToLoginVC() {
+        let loginVC = LoginViewController()
+        self.navigationController?.pushViewController(loginVC, animated: true)
+        loginVC.modalPresentationStyle = .fullScreen
     }
```

Comment:
```suggestion
    @objc fileprivate func goToLoginVC() {
        let loginVC = LoginViewController()
        self.navigationController?.pushViewController(loginVC, animated: true)
    }
```

`navigationController`의 `push` 화면 전환은 
`modalPresentationStyle`이 필요 없어 !